app/Actions/Database/StartDatabaseProxy.php

```
@@ -31,9 +31,22 @@ public function handle(StandaloneRedis|StandalonePostgresql|StandaloneMongodb|St
+                $containerName = "{$database->name}-{$database->service->uuid}";
+            } else {
+                if ($database->application_id) {
+                    $network = $database->application->destination->network;
+                    $server = data_get($database, 'application.destination.server');
+                    $containerName = "{$database->name}-" . generateApplicationContainerName($database->application);
+                } else {
+                    $preview = $database->application_preview;
+                    $network = $preview->application->destination->network;
+                    $server = data_get($database, 'application_preview.application.destination.server');
+                    $containerName = "{$database->name}-" . generateApplicationContainerName($preview->application, $preview->pull_request_id);
+                }
```

PHPMD (2.15.0)

[error] The method handle uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)